README.md

```diff
@@ -58,16 +58,23 @@ firectl \
   --metadata='{"foo":"bar"}'
 ```
 
-Getting Started on AWS
+Getting Started with Firectl
 ---
 
-- Create an `i3.metal` instance using Amazon Linux 2
+- Set up your machine as explained at https://github.com/firecracker-microvm/firecracker/blob/master/docs/dev-machine-setup.md
+- Install Go 1.11:
+
+  ```
+  sudo add-apt-repository ppa:longsleep/golang-backports
```

Comment:
Who maintains this PPA?  I think it'd be better to suggest installation from https://golang.org/dl/ given that we know those binaries are produced by the upstream project.